third_party/k8s.io/client-go/informers/externalversions/rbac/interface.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2017 Jetstack Ltd.
```

Comment:
❓ Wrong copyright header. Should be (c) Kuberneties authors I'd have thought.

Reply:
As I mentioned [here](https://github.com/jetstack/navigator/pull/82#discussion_r148257631) about generated code - I don't think it should be. The authors wrote the generators, I ran them. I depend on external code that *is* licensed by the Kubernetes authors, but the code in third_party is mine & I generated it. We depend on third party projects everywhere, but that doesn't mean those we depend on own the license on navigator.

> I'm not really sure to be honest - it's generated code, so I feel like it's not something that needs to be attributed to the k8s authors. If we were to change this to refer to the k8s authors, I feel like we should change all code generated by code-generators to be credited to the k8s authors.